tests/Integration/Behaviour/Features/Context/Domain/Product/Combination/UpdateCombinationFeatureContext.php

```diff
@@ -60,18 +60,6 @@ public function updateCombination(string $combinationReference, TableNode $table
         $this->updateCombinationStockAvailable($combinationReference, $tableNode->getRowsHash());
     }
 
-    /**
-     * @When I set combination ":combinationReference" as default
-     *
-     * @param string $combinationReference
-     */
-    public function setDefaultCombination(string $combinationReference): void
```

Comment:
Why did you remove this one? It's not used anymore? The default field can be changed from many place:
- combination form
- combination list
- combination bulk edition

But in any case I think all those cases can be handled via the unified `UpdateCombinationCommand` so I'm not sure we need the dedicated command anymore It should probably be removed in favor of the `UpdateCombinationCommand` no?